lib/test/unit/ui/console/testrunner.rb

```diff
@@ -408,7 +408,7 @@ def output_statistics(elapsed_time)
                          right_width,
                          slow_statistic[:elapsed_time],
                        ])
-                output(slow_statistic[:location])
+                output("--location %s" % slow_statistic[:location])
```

Comment:
We don't need to use format string here.

```suggestion
                output("--location #{slow_statistic[:location]}")
```

Reply:
Thanks for your suggestion!